CHANGELOG

```diff
@@ -1,11 +1,26 @@
 devel
 -----
 
-* Updated OpenSSL to 1.1.1j and OpenLDAP to 2.4.57.
+* Enable statistics in web UI in non-`_system` databases in cluster mode.
+  In cluster mode, the web UI dashboard did not display statistics properly
+  when not being logged in to the `_system` database. For all other databases
+  than `_system`, no statistics were displayed but just some "No data..."
+  placeholders.
+  Statistics for non-`_system` databases were not properly displayed since
+  3.7.6 due to an internal change in the statistics processing.
+
+  In addition, a new startup option `--server.statistics-all-databases`
+  controls whether cluster statistics are displayed in the web interface for
+  all databases (if the option is set to `true`) or just for the system
+  database (if the option is set to `false`).
+  The default value for the option is `true`, meaning statistics will be
+  displayed in the web interface for all databases. 
 
 * Added logging of elapsed time of ArangoSearch commit/consolidation/cleanup 
   jobs.
 
+* Updated OpenSSL to 1.1.1j and OpenLDAP to 2.4.57.
```

Comment:
I assume this is not related to this PR?

Reply:
No, it is unrelated. It just moves the most recent change to the top.